wearBenchmark/src/main/java/dev/johnoreilly/confetti/benchmark/ColdStartupBenchmark.kt

```diff
@@ -20,5 +20,6 @@ class ColdStartupBenchmark {
         startupMode = StartupMode.COLD
     ) {
         startActivityAndWait()
+        Thread.sleep(2000)
```

Comment:
does this not affect the time reported ?

Reply:
No, the metrics are generated inside the app, not in the benchmark test.